libs/counterfactuals/src/lib/CounterfactualChart.tsx

```diff
@@ -379,13 +379,14 @@ export class CounterfactualChart extends React.PureComponent<
   }
 
   private getCurrentClass(): string {
-    if (this.props.data.desired_class) {
-      return this.props.data.desired_class;
-    }
-    if (this.props.data.desired_range) {
-      return this.props.data.desired_range.join("->");
+    if (this.context.dataset.task_type === "regression") {
+      return (
+        (this.props.data.desired_range &&
```

Comment:
is it possible to add some kind of tooltip as well to explain what this means in the what if tool?  that can be a separate PR and maybe might need some PM input

Reply:
It could cause inconsistent if just add tooltip for regression cases, 
Also, the prefix "Current class" maybe need to rephrased, as in regression it's a range instead of specific class?
@minthigpen any idea?